packages/discovery-react-components/src/components/StructuredQuery/__tests__/StructuredQuery.test.tsx

```diff
@@ -52,6 +106,36 @@ describe('<StructuredQuery />', () => {
       });
     });
 
+    describe('when there are fields present', () => {
+      beforeEach(() => {
+        projectFields = [{ field: 'field name 1' }, { field: 'field name 2' }];
+        ({ structuredQuery } = setup({ projectFields }));
+      });
+
+      test('field dropdown has the correct default messages', () => {
+        const fieldDropdownTitleText = structuredQuery.getByText('Field');
+        const fieldDropdownPlaceholderText = structuredQuery!.getByPlaceholderText('Select field');
```

Comment:
why does this need `!.`?

Reply:
previously I was reseting `structuredQuery` to null after every test to avoid accidentally adding elements to an already rendered tree, which I since realized was unnecessary. While the value of `structuredQuery` could have been null, I needed to cast this, but I should be fine to remove this now thanks for the catch